src/lib/orm/migrations/1596269328762-V17_MigrateModeratorNonNull.ts

```diff
@@ -0,0 +1,16 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { CLIENT_ID } from '@root/config';
+
+export class V17MigrateModeratorNonNull1596269328762 implements MigrationInterface {
+
+	public async up(queryRunner: QueryRunner): Promise<void> {
+		await queryRunner.query(/* sql */`UPDATE public.moderation SET moderator_id = $1 WHERE moderator_id IS NULL;`, [CLIENT_ID]);
```

Comment:
```suggestion
		await queryRunner.query(/* sql */`UPDATE public.moderation SET moderator_id = '$1' WHERE moderator_id IS NULL;`, [CLIENT_ID]);
```

`moderator_id` is a varchar